.github/sync-files.yaml

```diff
@@ -0,0 +1,20 @@
+- repository: autowarefoundation/autoware
+  files:
+    - source: CODE_OF_CONDUCT.md
+    - source: CONTRIBUTING.md
+    - source: DISCLAIMER.md
+    - source: LICENSE
+    - source: .github/dependabot.yaml
+    - source: .github/workflows/automatic-rebase.yaml
+    - source: .github/workflows/pre-commit-optional.yaml
+    - source: .github/workflows/semantic-pull-request.yaml
+    - source: .github/workflows/spell-check.yaml
+    - source: .clang-format
+    - source: .markdown-link-check.json
+    - source: .markdownlint.yaml
+    - source: .pre-commit-config-optional.yaml
+    - source: .prettierignore
+    - source: .prettierrc.yaml
+    - source: .yamllint.yaml
+    - source: CPPLINT.cfg
+    - source: setup.cfg
```

Comment:
Don't you have to add some build-related workflows?

Reply:
update at https://github.com/tier4/ros2-project-template/pull/2/commits/c1cb583a00af7c06662225bbab09092b69ba8422